packages/cli/src/build/verify.rs

```diff
@@ -134,9 +134,19 @@ impl BuildRequest {
     /// will do its best to fill in the missing bits by exploring the sdk structure
     /// IE will attempt to use the Java installed from android studio if possible.
     pub(crate) async fn verify_android_tooling(&self, _rustup: RustupShow) -> Result<()> {
-        if self.krate.android_linker().is_none() {
+        let failed = 'a: {
+            let Some(ndk) = self.krate.android_ndk() else {
+                break 'a true;
+            };
+
+            let linker = self.build.target_args.arch().android_linker(&ndk);
+
+            !linker.exists()
+        };
+
+        if failed {
```

Comment:
this is weird code, can't we just use an unwrap_or or a .map or something that doesn't involve named breaks?